docker-compose.yml

```diff
@@ -293,6 +293,28 @@ services:
     networks:
       - tno
 
+  ####################### API #######################
+  ssh:
```

Comment:
This is just for testing, it doesn't need to be run unless you want an SSH server.